processes/roles.md

```diff
@@ -42,8 +40,26 @@ You can nominate someone for a role by emailing the chair of the TSC.
     in the TSC.
 
     Current members are the members of the Committers team on the seL4 GitHub
-    organisation. These include the TSC members and the kernel engineers as well
-    as the verification engineers of the Trustworthy Systems group.
+    organisation. These include all of the TSC members and the following
+    people:
+
+  - Ryan Barry, UNSW
+  - Scott Buckley, UNSW
+  - Peter Chubb, UNSW
+  - Axel Heider, Hensoldt Cyber
+  - Jashank Jeremy, UNSW
+  - Vincent Jackson, UNSW
+  - Zoltan Kocsis, UNSW
+  - Damon Lee, Kry10
+  - Michael McInerney, UNSW
+  - Johannes Åman Pohjohla, UNSW
+  - Miki Tanaka, UNSW
+  - Yanyan Shen, Cog Systems
```

Comment:
As far as I know, the affiliation has changed.

Reply:
Yes, @yyshen is now at NIO. And @zaklogician isn't at UNSW yet, but has accepted an offer

Reply:
Yes, good points, will add another PR. This mostly shows that having affiliations here is not such a good idea. It's too many people for this list to properly stay up to date.